client/src/app/result/evaluation/Evaluation.js

```diff
@@ -45,7 +45,7 @@ const Evaluation = ({ evaluations }) => (
                 <div className={`${item.name} score-bar-container`}>
                   <div
                     className="score-bar"
-                    style={{ width: `${calculateScore(item.total, item.max)}%` }}
+                    style={{width: `${calculateScore(item.total, item.max)}%`}}
```

Comment:
This causes the linter to complain. This was purposely added to standardise the styles of object notation.